src/webhooks/pubsub/stalebot.ts

```diff
@@ -3,10 +3,14 @@ import moment from 'moment-timezone';
 
 import {
   GETSENTRY_ORG,
+  SENTRY_MONOREPOS,
+  SENTRY_REPOS,
   STALE_LABEL,
   WAITING_FOR_COMMUNITY_LABEL,
 } from '@/config';
 
+const REPOS_TO_TRACK_FOR_STALEBOT = [...SENTRY_MONOREPOS, ...SENTRY_REPOS];
```

Comment:
Guess one question I have here, should we wait to roll this out to `sentry`? I'll put up PR's to remove `stale.yml` in repos before this goes in

Reply:
I'll make it `SENTRY_SDK_REPOS`